front/cypress/e2e/content_builder/content_builder_preview_toggle.cy.ts

```diff
@@ -84,16 +84,17 @@ describe('Content builder preview', () => {
     cy.get('#e2e-draggable-text').dragAndDrop('#e2e-content-builder-frame', {
       position: 'inside',
     });
+    cy.wait(5000);
     cy.get('#e2e-text-box').click();
```

Comment:
When I looked at the tests above, I saw that a wait was added. I think it has something to do with waiting for the preview. When I added it here the test passed consistently.